app/contexts/movies/queries/movie_queries.rb

```diff
@@ -0,0 +1,34 @@
+# Query Object
+module Contexts
+  module Movies
+    module Queries
+      class MovieQueries
+        attr_reader :movies
+
+        def initialize(movies = Movie.all)
+          @movies = movies
+        end
+
+        def all
+          @movies
+        end
+
+        def find_by(params)
+          @movies.find_by(params)
```

Comment:
you have `attr_reader :movies` so please use it
```suggestion
          movies.find_by(params)
```